src/Library.java

```diff
@@ -0,0 +1,74 @@
+public class Library {
+    private Book[] books;//Private array of book
+    private int i=0;//variable used in append method
+
+    public Library(Book[] books){ //Constructor to create library objects
+        this.books= books;  //array of book inputed as parameter saved into the declared but empty 
+    }
+    
+
+    public void append(Book book){//adds book to the array of books
+        books[i]=book;//adds book to the index of i which is the marks the next free space in the array
+        i++;//increases value of i so that it is 1 more than the index of the last book
+    }
+
+    public int getLastIndex(){//returns the next free space where a book can be added
+        return i;
+    }
+
+    public void decrement() {
+        i--;//When a book is removed this ensures the value of i is accurate
```

Comment:
this feels weird - I haven't seen the place this is used yet, but it feels like removing a book should happen as a method here, in this class, and the decrement should be part of that removing method